test/test_pebble.py

```diff
@@ -610,6 +639,110 @@ def test_service_equality(self):
         }
         self.assertEqual(plan.services, services_as_dict)
 
+    def test_plan_equality(self):
+        plan1 = pebble.Plan('''
+services:
+  foo:
+    override: replace
+    command: echo foo
+''')
+        self.assertFalse(plan1 == "foo")
+        plan2 = pebble.Plan('''
+services:
+  foo:
+    command: echo foo
+    override: replace
+''')
+        self.assertTrue(plan1 == plan2)
+        plan1_as_dict = pebble.PlanDict({
+            "services": {
+                "foo": pebble.ServiceDict({
+                    "command": "echo foo",
+                    "override": "replace",
+                })
+            },
+        })
+        self.assertTrue(plan1, plan1_as_dict)
```

Comment:
Was this intended as `plan1 == plan1_as_dict`?

Reply:
Yes - good catch, thanks! Another example of where pytest style `assert plan1 == plan1_as_dict` would be much better - I'm sure I wouldn't have typed `assert plan1, plan1_as_dict`.